rawkit/options.py

```diff
@@ -0,0 +1,405 @@
+""":mod:`rawkit.options` --- High level options for processing raw files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+import ctypes
+
+CLIP = 0
```

Comment:
There should be some encapsulation of constants like this so we don't completely pollute the `rawkit.options` namespace.